application.py

```diff
@@ -18,33 +18,75 @@
 app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
 db.init_app(app)
 
-
 with app.app_context():
     db.create_all()
 
 @app.route("/")
 def index():
-    return "<h1>Register</h1>"
+    if 'username' in session:
+        return redirect(url_for("home"))    
+    return redirect(url_for("register"))
 
 @app.route("/register")
 def register():
     return render_template("registration.html")
 
+@app.route("/logout/<username>")
+def logout(username):
+    session.pop(username, None)
+    return redirect(url_for('index'))
+
+
+@app.route("/home/<user>")
+def userHome(user):
+
+    if user in session:
+        return render_template("user.html", username=user)
+    
+    return redirect(url_for('index'))
+
+@app.route("/auth", methods =["POST", "GET"])
+def auth():
+
+    if request.method == "POST":
+
+        username = request.form.get('username')
+        usr_password = request.form.get('pwd')
+
+        userData = user.query.filter_by(username=username).first()
+
+        if userData is not None:
+            if userData.username == username and userData.password ==usr_password:
+                session[username] = username
+                return redirect(url_for('userHome', user = username))
+            else:
+                return render_template("registration.html", message = "Please enter correct username/password")
+        else:
+
+            return redirect(url_for('index'))
+    else:
+
+        return "<h1>Please login/register instead</h1>"
+
+
 @app.route("/userDetails",methods=["POST","GET"])
```

Comment:
As said earlier change this URI to /register and delete the /register written above

Reply:
K bro I will make changes based on the feedback